tests/general/discoverymode/docker_observer_discovery_test.go

```diff
@@ -127,8 +130,11 @@ func TestDockerObserver(t *testing.T) {
 							"config": map[string]any{
 								"collection_interval": "1s",
 								"labels": map[string]any{
-									"label_one": "${LABEL_ONE_VALUE}",
-									"label_two": "${LABEL_TWO_VALUE}",
+									"label_four":  "actual.label.four.value",
```

Comment:
I realize this has likely no functional impact at all, but is there a reason these labels aren't in order? Is the comparison somehow order sensitive and we're just trying to match what we get back from the collector container's initial config?

Reply:
[Maps are unordered](https://go.dev/ref/spec#Map_types) in go and [in yaml](https://yaml.org/spec/1.2.2/#3221-mapping-key-order) so no. The reason was they were declared alphabetically but I added `label_five` at the top w/o resorting.